content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: blog
+title: "CRI-O is moving towards pkgs.k8s.io"
+date: 2023-10-10
+slug: cri-o-community-package-infrastructure
+---
+
+**Author:** Sascha Grunert
+
+The Kubernetes community [recently announced](https://kubernetes.io/blog/2023/08/31/legacy-package-repository-deprecation/)
+that their legacy package repositories are frozen and that they will move to the
+[introduced community-owned packages](https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction) powered by the
+[openSUSE Build Service (OBS)](https://build.opensuse.org/project/subprojects/isv:kubernetes).
+CRI-O has a long history of utilizing [OBS for their package builds](https://github.com/cri-o/cri-o/blob/e292f17/install.md#install-packaged-versions-of-cri-o),
+but all of the packaging efforts have been done manually so far.
+
+The CRI-O community absolutely loves Kubernetes, which means that they're
+delighted to announce that:
+
+**All future CRI-O packages will be shipped as part of the officially supported
+Kubernetes infrastructure hosted on pkgs.k8s.io!**
+
+There will be a deprecation phase for the existing packages for sure, which is
```

Comment:
style nit: I think you could drop `for sure` here

Reply:
Fixed as suggested.